src/usr/local/www/diag_confbak.php

```diff
@@ -206,9 +206,18 @@
 print($form);
 
 if (is_array($confvers)) {
-	print_info_box(gettext('To view the differences between an older configuration and a newer configuration, ' .
-						   'select the older configuration using the left column of radio options and select the newer configuration in the right column, ' .
-						   'then press the "Diff" button.'));
+?>
+<div>
+	<div id="infoblock">
```

Comment:
It is the id="infobock" here that causes the block to be hidden and the "i" to appear.

To just change the color of the block (and not hide it) just add the bootstrap style as in:

print_Info_box("Some text for the box", "info"); or print_Info_box("Some text for the box", "success");